jujugui/static/gui/src/app/components/search-results/test-search-results.js

```diff
@@ -41,6 +41,36 @@ describe('SearchResults', function() {
     var output = shallowRenderer.getRenderOutput();
     assert.equal(output.props.className, 'search-results');
     var html = output.props.dangerouslySetInnerHTML.__html;
-    assert.isAbove(html.indexOf(query), -1);
+    assert.isAbove(html.indexOf("Loading"), -1);
+  });
+
+  it('search results load', function() {
```

Comment:
Oh, sorry, I mean these are supposed to read like a sentence. So at the moment it reads as "it search results load" instead of something like "it loads search results".